From 06045726d08e1f533f078447a4495e5a58341caa Mon Sep 17 00:00:00 2001
From: Simon Busch <morphis@gravedo.de>
Date: Fri, 2 Nov 2012 12:56:56 +0100
Subject: [PATCH 2/2] Create a shared library instead a static one

Open-webOS-DCO-1.0-Signed-off-by: Simon Busch <morphis@gravedo.de>
---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8677016..a05204d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -61,7 +61,7 @@ include_directories(include/public)
 file(GLOB SOURCE_FILES serviceinstall.cpp)
 
 # -- static library
-add_library(serviceinstaller ${SOURCE_FILES})
+add_library(serviceinstaller SHARED ${SOURCE_FILES})
 target_link_libraries(serviceinstaller 
                       ${GLIB2_LDFLAGS} 
                       ${GTHREAD2_LDFLAGS}
-- 
1.7.10.4

